static/app/views/settings/account/apiNewToken.spec.tsx

```diff
@@ -8,4 +10,66 @@ describe('ApiNewToken', function () {
       context: TestStubs.routerContext(),
     });
   });
+
+  it('renders with disabled "Create Token" button', async function () {
+    render(<ApiNewToken />, {
+      context: TestStubs.routerContext(),
+    });
+
+    expect(await screen.getByRole('button', {name: 'Create Token'})).toBeDisabled();
+  });
+
+  it('submits with correct scopes', async function () {
```

Comment:
maybe add a comment here or in the component that assigning admin scope will automatically give you read & write